### ISSUE TITLE
Select native multiple select values do not update

### ISSUE THREAD

**Issue by jupl** [stackworx/formik-material-ui#50]
```tsx
<Field
  native
  multiple
  {...props}
  component={Select}
/>
```

**Comment by cliedeman** [comment]
Something really weird is happening here. Some vague comments on some tickets say it has to do with our the event is triggered. Taking another look...

**Comment by cliedeman** [comment]
Created a fix in #86 but it requires react hooks...

**Comment by cliedeman** [comment]
Nvm, material ui 4 actually requires react 16.8.0 minimum

**Comment by cliedeman** [comment]
Released in v0.0.19
